### PR TITLE
Corrected typos and added a new config file

### PR DESCRIPTION
**Pull request by torrewk**
Corrected a typo in the box definition for SD. Another typo in the thermal force calculation for the BD was corrected. More debug information are printed now, in case overlaps between particles cause the simulation to stop.

### CODE REVIEW

File: files/gel_configuration.toml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ffwave = 6357442 #seed for far-field wave space contribution to thermal fluctuations
+ffreal = 6474524532 #seed for far-field real space contribution to thermal fluctuations
+nf = 325425435 #seed for near-field contribution to thermal fluctuations
+
+[output]
+store_stresslet = false
+store_velocity = false
+store_orientation = false
+writing_period = 10
+thermal_fluctuation_test = "none"  # Other options: far-field, lubrication #this should be 'none'
```
Comment (qubixes) [inline files/gel_configuration.toml:41]: This comment needs an update I think.

File: jfsd/main.py [hunk 2]
```diff
@@ -154,6 +154,14 @@ def main(Nsteps: int,
                  boundary_flag,thermal_test_flag,alpha_friction,ho_friction)
     return trajectory, stresslet_history, velocities, test_result
     
+def overlap_monitor(displacements_vector_matrix,N):
```
Comment (qubixes) [inline jfsd/main.py:157]:
```suggestion
def overlap_monitor(displacements_vector_matrix, N):
```

Comment (qubixes) [reply jfsd/main.py:157]: Perhaps you can type annotate as well?

Comment (qubixes) [inline jfsd/main.py:157]: Functions should generally be with a verb, so I might suggest `check_overlap` in this case, because that is what it does?

Comment (qubixes) [reply jfsd/main.py:157]: I see you also have a `check_overlaps` already. Perhaps you can rename that one to `find_overlaps` to distinguish better?

File: jfsd/main.py [hunk 3]
```diff
@@ -1029,13 +1027,9 @@ def update_positions(
...
             # check that thermal fluctuation calculation went well
             if ((not math.isfinite(stepnormff)) or ((n_iter_Lanczos_ff > 150) and (stepnormff >  0.003))):
-                overlaps = utils.check_overlap(
-                    displacements_vector_matrix, 2.)
-                if(overlaps > 0):
-                    print(
-                        'Warning! Particles are overlapping. Reducing the timestep might help prevent the simulation from stopping.')
-                    raise ValueError(
-                        f"Far-field Lanczos did not converge! Stepnorm is {stepnormff}, iterations are {n_iter_Lanczos_ff}. Overlapping pairs are {overlaps}. Eigenvalues of tridiagonal matrix are {diag_ff}. Abort!")
+                overlap_monitor(displacements_vector_matrix,N)
```
Comment (qubixes) [inline jfsd/main.py:1030]:
```suggestion
                overlap_monitor(displacements_vector_matrix, N)
```
And also for all the other occurrences.

**Review by qubixes — CHANGES_REQUESTED** [review]
Just some small comments from my side, otherwise it looks good.

File: jfsd/main.py (hunk 2 above)
Comment (torrewk) [reply jfsd/main.py:157]: Done now.

Comment (torrewk) [reply jfsd/main.py:157]: Modified as suggested.

File: jfsd/main.py (hunk 3 above)
Comment (torrewk) [reply jfsd/main.py:1030]: Modified.

**Pull request merged**